app/javascript/menu/main-menu.jsx

```diff
@@ -61,13 +63,27 @@ export const MainMenu = ({
     updateActiveItem(ManageIQ.redux.store.getState().router.location);
   }, []);
 
+  useEffect(() => {
+    function toggleExpanded(event) {
+      console.log("event.target",event.target);
+      if (event.key === 'Tab' && sidenavContainer.current && sidenavContainer.current.contains(event.target) && !event.target.classList.contains('menu-collapse-button')) {
+        setExpanded(true);
+      }
+    }
+    window.addEventListener('keydown', toggleExpanded);
+    return () => {
+      window.removeEventListener('keydown', toggleExpanded);
+    };
+  });
```

Comment:
Umm, what does this do? Why does it have a `console.log`?

Reply:
yes @skateman i added a line in description that there is a bug i will work on it and will remove console logs then.

Reply:
Sorry, I have a tendency of reading code first :see_no_evil: :disappointed_relieved:

Reply:
Remove the console.log